contracts/Colony.sol

```diff
@@ -93,6 +93,10 @@ contract Colony is ColonyStorage, PatriciaTreeProofs {
     return ColonyAuthority(address(authority)).hasUserRole(_user, _domainId, uint8(_role));
   }
 
+  function getUserRoles(address who, uint256 where) public view returns (bytes32) {
+    return DomainRoles(address(authority)).getUserRoles(who, where);
```

Comment:
Why not `ColonyAuthority`? `DomainRoles` is more specific but it seems more consistent to refer to `ColonyAuthority` everywhere for interacting with the permissions.